### ISSUE TITLE
[Feature request/suggestion] Acceleration for mouse or analog output

### ISSUE THREAD

**Issue by ren2r** [kozec/sc-controller#604]
Coming from using steam for a long time, there was some features that I was missing in this software, despite the fact there are some many more features here, I was missing the hipfire and acceleration.
I know this is not one of the most liked feature, but for the trackpads is good to have some customizable acceleration for the mouse output to compensate the size of the trackpads, since increasing the sensitivity using the sens modifier make small movements more harder to achieve. 
I was planning on trying to get do the implementation on my own, but thought about discussing it before.

**Comment by Veiranx** [comment]
Agreed!  While not absolutely necessary, it would be nice to quickly swipe the touch pad and move the cursor further than a slower swipe.
